src/bit-systems/networking.ts

```diff
@@ -36,11 +42,25 @@ export function takeOwnership(world, eid) {
   }
 }
 
-const createMessageDatas = new Map();
+type EntityID = number;
```

Comment:
Might be nice to start using this universally just for ease of reading. Tradeoff is tricking ourselves into thinking its something special and making EntityID specific things when we are actually just doing number stuff.

Reply:
It feels "correct" to call them `EntityID`s because (even though this isn't enforced), `EntityID`s are a narrower type than `number`s: They are (supposed to be) `uints` between `0` and `MAX_ENTITY_COUNT`.